Makefile

```diff
@@ -54,12 +60,18 @@ integration-static: .install.ginkgo # It needs to be release so we correctly tes
 	sudo -E "$(GOTOOLS_BINDIR)/ginkgo" $(TEST_FLAGS) $(GINKGO_FLAGS)
 
 .install.ginkgo:
-	GOBIN=$(abspath $(GOTOOLS_BINDIR)) go install github.com/onsi/ginkgo/v2/ginkgo@latest
+	GOBIN=$(abspath $(GOTOOLS_BINDIR)) go install github.com/onsi/ginkgo/v2/ginkgo@v$(GINKGO_VERSION)
```

Comment:
Let's move the 'v' into the GINKGO_VERSION, so there isn't any 'magic'.

Reply:
Done :+1: